README.md

```diff
@@ -125,6 +125,17 @@ Within the `Rollup__mdt` custom metadata type, add a new record with fields:
 
 You can have as many rollups as you'd like per object/trigger — all operations are boxcarred together for optimal efficiency.
 
+- `Description` (optional) - note-taking field in the event you'd like to provide additional info to other admins/users about your configured rollup
+
+#### Rollup Order By Custom Metadata
+
+Only valid to be tied to a given `Rollup__mdt` record with a `FIRST` or `LAST` Rollup Operation defined. Use these records to set up rules for how the records associated with your rollup will be ordered. Note that if a `FIRST` or `LAST` rollup is defined _without_ one of these records, the `Rollup Field On Calc Item` is used to order things.
+
+- `Ranking` - number value defining the order priority for any given `Rollup Order By` CMDT record. It's fine to start with 0 or with 1; ranking must be unique amongst all the records you create here
```

Comment:
I feel like this last part about Ranking isn't 100% accurate? Nothing technically breaks if 2+ records have the same Ranking (as far as the CMDT is concerned), and [you have `DeveloperName` as a secondary sort](https://github.com/jamessimone/apex-rollup/blob/a5c6934cdfbcb5e7d4ed7a91ebc462bcf8587a4c/rollup/core/classes/Rollup.cls#L1524). 

Would it be more accurate to say something like "ranking should be unique amongst all the records you create here for the same parent `Rollup__mdt` record. When 2 (or more) `Rollup Order By` CMDT records associated with the same `Rollup__mdt` record have the same `Ranking`, then Apex Rollup also sorts by `DeveloperName`"? 

If you want to keep the text as-is, that 100% makes sense too as far as motivating admins/devs to properly configure the `Ranking`, but just thought I'd bring it up.

Reply:
I made it a unique field so as far as records that are actually created (rather than spun up in Apex), that field-level integrity will be enforced. While I don't have anything actually checking past the point where these records of queried for that ranking is unique, I'd prefer to _act_ like the only possible choice is to have ranking be unique (and pre-sorted)

Reply:
Oooooh, got it! I overlooked that you had marked the `Ranking` field as unique. Ignore me! 🤐